pluginlib/include/pluginlib/class_loader_imp.hpp

```diff
@@ -423,14 +424,6 @@ std::vector<std::string> ClassLoader<T>::getAllLibraryPathsToTry(
     package_prefix + path_separator + "bin" + path_separator + exporting_package_name,
   };
 
-  // Prepare to setup the relative file paths.
```

Comment:
Can you please add to the commit message, PR description, or both information about the removal of this block and either where the code was moved or why it can be removed.

Reply:
I removed the `debug_mode` variable because poco was returning the suffix in case we are in debug mode or not. I was trying to fix this fast. I will dig more

Reply:
If https://github.com/ros/pluginlib/pull/186/files#diff-83eae4c2eef591307fdaf5e9d11dc2c1L427 was always true previously, that's sufficient reasoniong to drop the conditional. While it'd be interesting to know why I think it's academic.

Reply:
can I do somethin like ?

```
bool debug_mode;
#if defined(_DEBUG)
	debug_mode = true;
#else
	debug_mode = false;
#endif
```

Reply:
I don't know what poco was doing behind the scenes which the previous check was piggybacking on. It might be worth introducing such a check into rc{,pp}utils in order to properly replace our usage of it.